Maestro/subscriptions.json

```diff
@@ -666,6 +666,39 @@
         }
       }
     },
+    // Automate merging aspnetcore release/7.0-rc branches back to release/7.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/aspnetcore/blob/release/7.0-rc/**/*"
```

Comment:
Will this try to merge release/7.0 into release/7.0❔ If yes, does that matter❔

Reply:
I don't think so, it's a direct copy of what runtime does & what we did last year